### ISSUE TITLE
current generating devices cannot be connected to binary neurons

### ISSUE THREAD

**Issue by tobikausk** [nest/nest-simulator#332]
The ac-generator (and probably also dc- and step-current-generator) cannot be connected to binary neurons. This would be a useful feature (as you sometimes want to create "time varying thresholds") and would not create any inconsistencies.
The problem can be fixed rather easily and I will open a PR soon.

**Comment by heplesser** [comment]
@mhelias What do you think about this proposal? Binary neurons are quite different from everything else, so interpreting "current" input as a signal that represents a time-varying threshold is probably something one could do. But documentation would have to be very clear.

**Comment by mhelias** [comment]
Tobias and I discussed this. Actually, the connection of ac and dc denerators to binary neurons only fails since we implemented a stricter checking for compatibility of spiking and binary models. In previous revisions this has been possible and some of our published articles rely on this feature. The only thing to implement is the function send_signal() in all generators that should be compatible. The implementation of the binary neurons is already prepared to handle dc_input as an additive signal to the total input (thus equivalent to varying threshold). Documentation of the feature needs to be checked.
